test/Core.Test/Tokens/TestTokenable.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Bit.Core.Tokens;
+
+namespace Bit.Core.Test.Tokens
+{
+    public class TestTokenable : Tokenable
+    {
+        public bool ForceInvalid { get; set; } = false;
+        public override bool Valid
+        {
+            get => !ForceInvalid;
+        }
+    }
```

Comment:
Same this always seems to return true on `Valid`.

Reply:
This one is probably useless, yep. I'll remove